src/index.js

```diff
@@ -27,9 +27,11 @@ const questions = {
 };
 
 commander
-	.option( "-r, --release [type]", "Release type (major, minor, patch)", /^(major|minor|patch)/i )
+	.option( "-r, --release [type]", "Release type (major, minor, patch, premajor, preminor, prepatch, prerelease)", /^(major|minor|patch|premajor|preminor|prepatch|prerelease)/i )
```

Comment:
Seems to be working as intended with an added check, but would be good to run through all the scenarios in the review!